### DISCUSSION TITLE
SOA expire behaviour on secondary for MySQL backend

### DISCUSSION THREAD

**Discussion by dransome** [PowerDNS/pdns#14682] [Q&A] [answered]
I couldn't find confirmation in documentation or code, so I'm hoping someone can point me in the right direction...

Does the MySQL backend also ignore SOA expire time on secondary just [as was mentioned for the bind backend](https://github.com/PowerDNS/pdns/issues/11185#issuecomment-1012288326)?

**Comment by zeha** [comment, marked as answer]
The expire field is not used, regardless of the backend.
